config_example.yaml

```diff
@@ -475,4 +475,9 @@ openobserve:
   # username: "a" # use this username to authenticate to OpenObserve if the username is not empty (default: "")
   # password: "" # use this password to authenticate to OpenObserve if the password is not empty (default: "")
   # customHeaders: # Custom headers to add in POST, useful for Authentication
-  #   key: value
\ No newline at end of file
+  #   key: value
+
+dynatrace:
+  apitoken: "" # Dynatrace API token with the "logs.ingest" scope, more info : https://dt-url.net/8543sda
+  apiurl: "" # Dynatrace API url, use https://ENVIRONMENTID.live.dynatrace.com/api for Dynatrace SaaS and https://YOURDOMAIN/e/ENVIRONMENTID/api for Dynatrace Managed, more info : https://dt-url.net/ej43qge
+  # minimumpriority: "" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
```

Comment:
the `checkcert ` is missing here but declared in the struct and at client creation